integrationtests/volume_test.go

```diff
@@ -196,14 +111,25 @@ func negativeVolumeTests(t *testing.T) {
 	runNegativeMountVolumeRequest(t, client, "", "")
 	runNegativeMountVolumeRequest(t, client, "-1", "")
 
-	// Dismount volume negative tests
-	runNegativeDismountVolumeRequest(t, client, "", "")
-	runNegativeDismountVolumeRequest(t, client, "-1", "")
+	// Unmount volume negative tests
+	runNegativeUnmountVolumeRequest(t, client, "", "")
+	runNegativeUnmountVolumeRequest(t, client, "-1", "")
 
 	runNegativeVolumeStatsRequest(t, client, "")
 	runNegativeVolumeStatsRequest(t, client, "-1")
 }
 
+// sizeIsAround returns true if the actual size is around the expected size
+// (considers the fact that some bytes were lost)
+func sizeIsAround(t *testing.T, actualSize, expectedSize int64) bool {
+	// An upper bound on the number of bytes that are lost when creating or resizing a partition
+	var volumeSizeBytesLoss int64 = (20 * 1024 * 1024)
```

Comment:
how this volumeSizeBytesLoss is obtained? Is there some doc related to this?

Reply:
I got this value from looking at the GetStats output after you format a partition, if you format it to 1GB you'r getting actually ~0.9844GB, probably @ddebroy might help us understanding where are those bytes going :)

Anyways if you do 1GB - 20MB that's around 0.98GB which I thought it was a good lower bound